sci-libs/punc/punc-1.5-r1.ebuild

```diff
@@ -37,6 +37,7 @@ BDEPEND="
 PATCHES=(
 	"${FILESDIR}"/${PV}-linking.patch
 	"${FILESDIR}"/1.4-doc.patch
+	"${FILESDIR}"/${P}-explicit-main.patch
```

Comment:
Has this patch been sent upstream?

Reply:
Just sent it to author Michael Holst.

Reply:
My email delivery failed, it may be bad address in package site.